packages/kotti-ui/source/kotti-card/KtCard.vue

```diff
@@ -51,23 +45,21 @@ export default defineComponent<KottiCard.PropsInternal>({
 .kt-card {
 	display: flex;
 	flex-direction: column;
-	word-break: break-all;
+	padding: var(--unit-4);
+	word-break: break-word;
 	background: var(--ui-background);
-	border: var(--unit-q) solid var(--ui-02);
+	border: 1px solid var(--ui-02);
 	border-radius: var(--border-radius);
 
+	&--has-top-image {
+		padding-top: 0;
+	}
+	&--has-bottom-image {
+		padding-bottom: 0;
+	}
+
 	&__header {
 		order: 2;
-
-		h1,
-		h2,
-		h3,
-		h4,
-		h5,
-		h6 {
-			margin-top: 0;
```

Comment:
just to clarify, this was removed because we shouldn't control the padding of deeper components, it's the decision of the user app